src/org/opensolaris/opengrok/web/AuthorizationFilter.java

```diff
@@ -35,6 +35,7 @@
 import javax.servlet.http.HttpServletResponse;
 import org.opensolaris.opengrok.configuration.Project;
 import org.opensolaris.opengrok.logger.LoggerFactory;
+import org.opensolaris.opengrok.web.stats.StatisticsFilter;
```

Comment:
?

Reply:
It was added by Idea because of the refactoring. You can see there is a java doc without full name. Better to change the javadoc. I will repair it.

Reply:
ok